integrations/localhero/src/main/java/ch/stayhome/integrations/localhero/job/LocalHeroBatchConfig.java

```diff
@@ -0,0 +1,138 @@
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.job.builder.FlowBuilder;
+import org.springframework.batch.core.job.flow.Flow;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.batch.core.launch.support.RunIdIncrementer;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.task.SimpleAsyncTaskExecutor;
+import org.springframework.scheduling.annotation.Scheduled;
+
+@Configuration
+@EnableConfigurationProperties({LocalHeroProperties.class})
+@Slf4j
+public class LocalHeroBatchConfig {
+
+	private static final int SKIP_LIMIT = 10;
+
+	private final LocalHeroProperties localHeroProperties;
+
+	private final JobBuilderFactory jobBuilderFactory;
+
+	private final StepBuilderFactory stepBuilderFactory;
+
+	private final JobLauncher jobLauncher;
+
+	public LocalHeroBatchConfig(final LocalHeroProperties localHeroProperties, final JobBuilderFactory jobBuilderFactory, final StepBuilderFactory stepBuilderFactory, final JobLauncher jobLauncher) {
+		this.localHeroProperties = localHeroProperties;
+		this.jobBuilderFactory = jobBuilderFactory;
+		this.stepBuilderFactory = stepBuilderFactory;
+		this.jobLauncher = jobLauncher;
+	}
+
+	@SneakyThrows
+	@Scheduled(cron = "${stayhome.integrations.localhero.scrape-cron}")
+	public void perform() {
+		log.info("Import started at {}", new Date());
+
+		JobParameters params = new JobParametersBuilder()
+				.addString("ID", String.valueOf(System.currentTimeMillis()))
+				.toJobParameters();
+
+		jobLauncher.run(importLocalHeroJob(), params);
+	}
+
+	@Bean
+	public Job importLocalHeroJob() {
+		return jobBuilderFactory.get("localHeroImportJob")
+				.incrementer(new RunIdIncrementer())
+				.start(this.parallelFlow())
+				.build();
+	}
+
+	private Step parallelFlow() {
+		final List<SourceConfig> sourceConfigs = this.localHeroProperties.getSources();
+		return this.stepBuilderFactory.get("importing-steps")
+				.flow(new FlowBuilder<Flow>("importing-flows")
+						.split(this.parallelTaskExecutor())
+						.add(sourceConfigs.stream()
+								.map(this::stepFlow)
+								.toArray(Flow[]::new))
+						.build())
+				.build();
+	}
+
+	private Flow stepFlow(SourceConfig sourceConfig) {
+		return new FlowBuilder<Flow>("flow-" + sourceConfig.getKey())
+				.start(step(sourceConfig))
+				.build();
+	}
+
+	private Step step(SourceConfig sourceConfig) {
+		final LocalHeroChApi localHeroChApi = this.buildApi(sourceConfig);
+		return stepBuilderFactory.get("import-step-" + sourceConfig.getKey())
+				.<LocalHeroPost, StayHomeEntry>chunk(localHeroProperties.getChunkSize())
+				.faultTolerant()
```

Comment:
@lugipfupf we don't want to job the fail if a single entry failed -> let's skip up to X entries that are allowed to fail till the batch-job is considered as failed